packages/build/src/tarball.ts

```diff
@@ -0,0 +1,130 @@
+import tar from 'tar';
+import path from 'path';
+import AdmZip from 'adm-zip';
+import pkgDeb from 'pkg-deb';
+import BuildVariant from './build-variant';
+
+/**
+ * Get the path to the tarball.
+ *
+ * @param {string} outputDir - The output directory.
+ * @param {string} platform - The platform.
+ * @param {string} version - The version.
+ *
+ * @returns {string} The path.
+ */
+export const tarballPath = (outputDir: string, platform: string, version: string): string => {
+  if (platform === BuildVariant.Linux) {
```

Comment:
should we rename platform to buildVariant here?

Reply:
yes!